CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+# v1.59.0
+
+:new: What's new:
+
+- UI: Overhaul lakeFS look & feel (#9022)
+
+:bug: Bugs fixed:
+
+- Fix GC failure on unknown files under the _lakefs folder (#9119)
```

Comment:
Not sure this belongs in the lakeFS changelog. This change is in the GC client though we don't really have a release for it